lib/stm32/common/st_usbfs_core.c

```diff
@@ -79,26 +88,39 @@ void st_usbfs_ep_setup(usbd_device *dev, uint8_t addr, uint8_t type,
 	USB_SET_EP_TYPE(addr, typelookup[type]);
 
 	if (dir || (addr == 0)) {
+		// TODO support isochronous endpoints with their double buffering
+
```

Comment:
Adding the todo seems odd when that's what thhis PR is adding?

Reply:
Should've added "WIP" to this PR. I've only implemented it for IN endpoints. This is where OUT implementation should go.